docker/dev/nginx/nginx.conf

```diff
@@ -54,19 +54,31 @@ http {
     }
 
     location /api/geolocation/ {
-      proxy_pass http://geolocation:7070/;
+      proxy_pass http://geolocation:7070/service_geo_location/;
+    }
+
+    location /api/geolocation/v1/ {
+      proxy_pass http://geolocation:7070/service_geo_location/;
     }
 
     location /api/thingaggregate/ {
-      proxy_pass http://thingaggregate:7070/;
+      proxy_pass http://thingaggregate:7070/service_thing_aggregate/;
     }
 
     location /api/actor/ {
-      proxy_pass http://actor:7070/;
+      proxy_pass http://actor:7070/service_actor/;
+    }
+
+    location /api/actor/v1/ {
+      proxy_pass http://actor:7070/service_actor/;
```

Comment:
Only the storagefacility service should have a v1 mapping enabled. The only reason is for the Delphi client not to break.